remediation/code-samples/01-countEvens.js

```diff
@@ -11,7 +11,9 @@
  * @param {number[]} array - An array of numbers
  * @returns {number} The count of even integers in the array
  */
-function countEvens(array) {
+
+// PLEASE REVIEW
+ function countEvens(array) {
   for(i=0;i<array.length;i++)
   {
     if(array[i]%2 != 0)
```

Comment:
The goal of this function is to return a count of the even integers.

We _DO_ use the modulo (%) operator to check if numbers are even or odd. 
If` x%2 == 0`, that means there is NO remainder when we divide by 2 (and the number is even). If `x%2 != 0`, there IS a remainder when we divide by 2 (and the number is odd).

But which question are you asking with this line of code...
- Is the element at array[i] **even**? OR
- Is the element at array[i] **odd**?